youtube_dl/extractor/ximalaya.py

```diff
@@ -0,0 +1,184 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
```

Comment:
This blank line is unnecessary, and import statements for built-in modules should be sorted alphabetically.